packages/react/src/generators/cypress-component-configuration/schema.json

```diff
@@ -37,6 +37,13 @@
       "type": "boolean",
       "description": "Skip formatting files",
       "default": false
+    },
+    "bundler": {
+      "description": "The bundler to use.",
```

Comment:
maybe we don't need this if we can tell what project it's being added to we can see if that project uses vite or webpack?